demo/src/app.js

```diff
@@ -88,10 +87,7 @@ const AppContent = () => {
 
     // Can't use lazy initializer because useRouteMatch is a hook
     const [initialMatchSilentRenewCallbackUrl] = useState(
-        useRouteMatch({
-            path: '/silent-renew-callback',
-            exact: true,
-        })
+        location.pathname === '/silent-renew-callback'
```

Comment:
You can't use location.pathname directly because it contains the basename. Using the react router api allows to abstract it away because it's defined in the root Router